### PR TITLE
chore: update escargot with bug fix

### PR DESCRIPTION
**Pull request by clover2123**

### CODE REVIEW

**Comment by haesik** [comment]
is it still valid PR?

**Comment by clover2123** [comment]
@haesik it's still in progress, I'll update it as soon as faster.

File: escargot.gyp [hunk 1]
```diff
@@ -47,6 +47,7 @@
       'escargot_configs': [
         '-DESCARGOT_SMALL_CONFIG=1',
         '-DESCARGOT_USE_CUSTOM_LOGGING=ON',
+        '-DESCARGOT_USE_EXTENDED_API=ON',
```
Comment (clover2123) [inline escargot.gyp:50]: Escargot build option updated.
To use specific non-standard features of Escargot like Template, build option `ESCARGOT_USE_EXTENDED_API` should be set ON.

File: deps/node/test/skip_tests.txt [hunk 2]
```diff
@@ -193,7 +194,9 @@ test/parallel/test-worker-nexttick-terminate.js
 test/parallel/test-worker-process-cwd.js
 test/parallel/test-worker-process-env.js
 test/parallel/test-worker-sharedarraybuffer-from-worker-thread.js
+test/parallel/test-worker-stack-overflow.js
 test/parallel/test-worker-stdio.js # CI 테스트중 간헐적으로 crash. 타이밍 이슈로 추정됨
+test/parallel/test-worker-stdio-from-preload-module.js # occasionally crashed in CI, might be timing issue
```
Comment (clover2123) [inline deps/node/test/skip_tests.txt:199]: Several tests failed occasionally in CI. 
But when I run these tests on my local environment, it passed.
I think that this is related with timing issue as these tests all employ `worker` method.

**Review by daeyeon — APPROVED** [review]

**Review by hs0225 — APPROVED** [review]

**Pull request merged**